index.html

```diff
@@ -0,0 +1,151 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Survey Form</title>
+</head>
+<body>
+    <div class="header">
+        <h1 id="title">Free CodeCamp Survey Form</h1>
+        <p id="description"> <i> Thank you for taking the time to help us improve the platform</i></p>
+    
+    </div>
+    <div class="container">
+
+
+    
+    <form id="survey-form">
+        <label for="name">Name
```

Comment:
Kindly remove extra lines to improve the professional formatting of your html file. Doing this will make your code look less cluttered and more clean and readable for other programmers too👌😊